server/swap/swap_test.go

```diff
@@ -1934,27 +1898,14 @@ func TestCancel(t *testing.T) {
 	}
 }
 
-func TestState(t *testing.T) {
+/*
```

Comment:
Are you keeping these tests around to adapt them?

Reply:
Nope.  They're trash too.